RCAIDE/Methods/Aerodynamics/Airfoil/compute_airfoil_properties_from_polar_files.py

```diff
@@ -0,0 +1,189 @@
+## @ingroup Methods-Aerodynamics-Airfoil
+# compute_airfoil_properties_from_polar_files.py
+# 
+# Created:  Oct 2023, Racheal M. Erhard
+# Modified: 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+import Legacy.trunk.S as SUAVE
```

Comment:
Try not to use the name SUAVE. Hence why it is S

Reply:
Fixed